Components/text-utilities/text.css

```diff
@@ -0,0 +1,115 @@
+@import url(../Shared/reset.css);
+
+.heading-1,
+.heading-2,
+.heading-3,
+.heading-4,
+.heading-5,
+.heading-6,
+.text-subtitle,
+.text-body-lg,
+.text-body-sm,
+.text-caption,
+.text-gutterBottom,
+.text-noWrap,
+.text-align-left,
+.text-align-right,
+.text-align-center {
+    font-family: "Quicksand", sans-serif;
```

Comment:
if possible you can give font to universal selector which could reduce the number of classes